### DISCUSSION TITLE
CRDT as Datatype

### DISCUSSION THREAD

**Discussion by vladimirmyshkovski** [vlang/v#15062] [Ideas]
I saw [datatypes](https://github.com/vlang/v/tree/1546645f637c00e4d13417efe6ee5501684834cc/vlib/datatypes) and at the same time, I am working on an implementation of [CRDT](https://en.wikipedia.org/wiki/Conflict-free_replicated_data_type) in V. 

All types are implemented in this [repository](https://github.com/vladimirmyshkovski/crdt). 

Is `CRDT` of interest for `Datatypes`?

**Comment by Hunam6** [comment]
At a first glance, yes, I think so, as it's described as a "data structure". However I don't know this data type.

**Comment by vladimirmyshkovski** [comment]
@Hunam6 
> At a first glance, yes, I think so, as it's described as a "data structure". However I don't know this data type.

This is a datatypes for conflict-free data merging in distributed systems.

**Comment by medvednikov** [comment]
Yes, sounds good. Thanks :)
